src/shared/components/data-table/components/table-controls/components/table-actions/TableActions.scss

```diff
@@ -0,0 +1,3 @@
+.simpleSelect {
+  --select-min-width: 20ch;
+}
```

Comment:
Why do we need this? Not just trusting Select to adapt to the width of its longest option?

Reply:
Removed this. I remember adding is to fix the wrapping issue on small screens. I don't think it is still needed.